policy/aggregation_type_options.go

```diff
@@ -132,45 +89,49 @@ type AggregationTypesOptions interface {
 
 	/// Write-only options.
 
-	// SetCounterSuffixOverrides sets the overrides for counter suffixes.
-	SetCounterSuffixOverrides(m map[AggregationType][]byte) AggregationTypesOptions
+	// SetTypeStrings sets the global type strings. TypeStringTransformerFn
+	// will be applied to these type strings.
+	SetTypeStrings(m map[AggregationType][]byte) AggregationTypesOptions
```

Comment:
nit: `SetGlobalTypeStringOverrides`? Perhaps also update the comment to reflect this is the global override?